src/components/Notification/InactivityMessage.jsx

```diff
@@ -67,76 +72,44 @@ const InactivityMessage = () => {
   return (
     showPopup &&
     activeUser && (
-      <StyledOverlay>
-        <StyledModal>
-          <StyledContainer>
-            <p>You have been inactive for a few minutes now. Would you like to log out?</p>
-            <ButtonsContainer>
-              <StyledButton onClick={() => setShowPopup(false)}>Continue Session</StyledButton>
-              <LogoutButton>
-                <StyledLogoutButton onClick={handleLogout}>Log Out</StyledLogoutButton>
-              </LogoutButton>
-            </ButtonsContainer>
-          </StyledContainer>
-        </StyledModal>
-      </StyledOverlay>
+      <Dialog
+        open={showPopup}
+        onClose={() => setShowPopup(false)}
+        aria-labelledby="inactivity-message-title"
+        aria-describedby="inactivity-message-description"
+      >
+        <DialogTitle id="inactivity-message-title">Continue session?</DialogTitle>
+        <DialogContent id="inactivity-message-description">
+          <DialogContentText>
+            You have been inactive for a while now. Would you like to continue using PASS?
+          </DialogContentText>
+        </DialogContent>
+        {/* TODO: In future PR, add countdown timer to automatically log user out if they do not select continue */}
+        {/* e.g. "You will be automatically logged out in 5:00 minutes" */}
+        <DialogActions>
+          <LogoutButton>
+            <Button
+              variant="outlined"
+              color="error"
+              endIcon={<LogoutIcon />}
+              onClick={handleLogout}
+            >
+              Log Out
+            </Button>
+          </LogoutButton>
+          <Button
+            variant="contained"
+            color="success"
```

Comment:
we should probably change this to match all the other modal/green colors.  Or we could change the rest to match this I guess.